packages/dht/src/dht/store/StoreRpcLocal.ts

```diff
@@ -54,13 +59,30 @@ export class StoreRpcLocal implements IStoreRpc {
         const dataEntry = request.entry!
         const wasStored = this.config.localDataStore.storeEntry(dataEntry)
         if (wasStored) {
-            this.config.replicateDataToNeighbors((context as DhtCallContext).incomingSourceDescriptor!, request.entry!)
+            this.replicateDataToNeighbors((context as DhtCallContext).incomingSourceDescriptor!, request.entry!)
         }
         const key = getDhtAddressFromRaw(dataEntry.key)
-        if (!this.config.isLocalNodeStorer(key)) {
+        if (!this.isLocalNodeStorer(key)) {
             this.config.localDataStore.setAllEntriesAsStale(key)
         }
         logger.trace('server-side replicateData() at end')
         return {}
     }
+
+    private replicateDataToNeighbors(requestor: PeerDescriptor, dataEntry: DataEntry): void {
+        const dataKey = getDhtAddressFromRaw(dataEntry.key)
+        const storers = this.config.getStorers(dataKey)
+        const selfIsPrimaryStorer = areEqualPeerDescriptors(storers[0], this.config.localPeerDescriptor)
```

Comment:
If we nowadays use term `localNode` instead of `self`?

```suggestion
        const isLocalNodePrimaryStorer = areEqualPeerDescriptors(storers[0], this.config.localPeerDescriptor)
```